python_modules/libraries/dagster-airlift/kitchen-sink/kitchen_sink_tests/integration_tests/test_api.py

```diff
@@ -0,0 +1,21 @@
+from dagster_airlift.core.airflow_instance import AirflowInstance
+from dagster_airlift.core.basic_auth import AirflowBasicAuthBackend
+from kitchen_sink.airflow_instance import (
+    AIRFLOW_BASE_URL,
+    AIRFLOW_INSTANCE_NAME,
+    PASSWORD,
+    USERNAME,
+)
+
+
+def test_configure_dag_list_limit(airflow_instance: None) -> None:
+    """Test that batch instance logic correctly retrieves all dags when over batch limit."""
+    af_instance = AirflowInstance(
+        auth_backend=AirflowBasicAuthBackend(
+            webserver_url=AIRFLOW_BASE_URL, username=USERNAME, password=PASSWORD
+        ),
+        name=AIRFLOW_INSTANCE_NAME,
```

Comment:
might be worth checking that we call the endpoint the appropriate amount of times, too (e.g. that the limit as actually being applied)